displaylink.run

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+PATH=/usr/lib/displaylink
+# sv check <display-manager> ?
```

Comment:
Is this the equivalent of `WantedBy=graphical.target` in systemd?

See jspaces comment on the [Artix Forum](https://forum.artixlinux.org/index.php/topic,5038.msg32286.html#msg32286)